features/system_specs/system_specs.feature

```diff
@@ -1,13 +1,30 @@
 Feature: System spec
 
     System specs are RSpec's wrapper around Rails' own
-    [system tests](http://guides.rubyonrails.org/testing.html#system-testing).
-    We encourage you to familiarize yourself with their documentation.
-
-    RSpec **does not** use your `ApplicationSystemTestCase` helper. Instead it uses
-    the default `driven_by(:selenium)` from Rails. If you want to override this
-    behaviour you can call `driven_by` manually in a test.
-
+    [system tests](https://guides.rubyonrails.org/testing.html#system-testing).
+    
+    > System tests allow you to test user interactions with your application,
+    > running tests in either a real or a headless browser. System tests use
+    > Capybara under the hood.
+    >
+    > By default, system tests are run with the Selenium driver, using the
+    > Chrome browser, and a screen size of 1400x1400. The next section explains
+    > how to change the default settings.
+
+    System specs are marked by `:type => :system` or if you have set
+    `config.infer_spec_type_from_file_location!` by placing them in
+    `spec/system`.
+
+    The Capybara gem is automatically required, so you need that in your
+    Gemfile. Configure a webserver (e.g. `Capybara.server = :webrick`) before
```

Comment:
Does Rails require you to add it to your `Gemfile`? I think this needs clarifying that this is a Rails requirement.

Reply:
You're absolutely right, `rails new` adds these to the `test` group, no need to clutter with that part.

Reply:
>     The Capybara gem is automatically required, and Rails includes it in the
>     Gemfile of generated applications. Configure a webserver (e.g.
>     `Capybara.server = :webrick`) before attempting to use system specs.
>

Reply:
How does the current wording in the PR sit with you, now?